sigs.yml

```diff
@@ -466,6 +466,20 @@
     gcLiaison:
     - name: 'Juraci Paixão Kröhling'
       github: jpkrohling
+  - name: 'GoLang: Compile Time Instrumentation'
+    meeting: Every other Thursday at 08:00 UTC
+    notes:
+      type: gDoc
+      value: 1XkVahJfhf482d3WVHsvUUDaGzHc8TO3sqQlSS80mpGY
+    chat:
+      - type: slack
+        name: '#otel-go-compile-time-instr'
```

Comment:
can we align slack name, repo name and team names? e.g.

either

* `https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation` 
* `#otel-go-compile-instrumentation`
* `@go-compile-instrumentation-maintainers `

or

* `https://github.com/open-telemetry/opentelemetry-go-compile-time-instr` 
* `#otel-go-compile-time-instr`
* `@go-compile-time-instr-maintainers `

Reply:
Channel renamed and PR updated